repository/index.js

```diff
@@ -0,0 +1,21 @@
+import mu from 'mu';
+
+const mandateeIsCompetentOnFutureAgendaItem = async (endDateOfMandee, mandateeId) => {
+  const query = `
+  PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+
+  ASK WHERE {
+  
+  ?procedurestap besluitvorming:heeftBevoegde <http://kanselarij.vo.data.gift/id/mandatarissen/${mandateeId}> .
```

Comment:
Het is good practice om van elke entiteit die je in een query aanraakt het rdf type af te dwingen. Bijvoorbeeld
```
?procedurestap a dbpedia:UnitOfWork
```
Op die manier vermijd je van per ongeluk totaal andere dingen dan procedurestappen te selecteren, die ook relaties onder het predicaat `besluitvorming:heeftBevoegde` hebben. In praktijk zal voorgaande met onze dataset weinig voorvallen, maar als er types van entiteiten in data veranderen, en je bent deze code vergeten reviseren in het licht van zo'n migratie, zal het probleem snel duidelijk worden. Zonder de types te specifiëren kan je veel subtielere bugs hebben.

(zie de api configuratie voor de verschillende types, bv [procedurestap](https://github.com/kanselarij-vlaanderen/kaleidos-project/blob/eaacf6d30fc6b8e2c4a37f26dcb0e3dc1c03e48b/config/resources/dossier-domain.lisp#L74)

Reply:
Checks toegevoegd in de query